packages/@react-spectrum/s2/stories/Tooltip.stories.tsx

```diff
@@ -80,7 +80,9 @@ const ExampleRender = (args: any) => {
   );
 };
 
-export const Example = {
+type Story = StoryObj<typeof TooltipTrigger>;
```

Comment:
Looks like our story can handle the combination of TooltipTrigger and Tooltip props